<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>添加操作</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_op_type">
       <item>
        <property name="text">
         <string>执行命令</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>快捷键</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>运行程序</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_extra_data"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_gesture"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_op_gesture">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前操作与手势配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="table_config"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
